### DISCUSSION TITLE
Audio stutter after a while

### DISCUSSION THREAD

**Discussion by hsshhhss** [jamulussoftware/jamulus#1946] [Technical Support & Help]
Audio starts to stutter like crazy every 5 to 30 minutes. Ping, Delay, and Jitter are all with green light. 

Not sure how to reproduce it but it happens to me all the time.

macOS Big Sur 11.0.1
Jamulus 3.8.0 installed using dmg file, downloaded from the official website
Audio input is routed from a Loopback virtual channel. The input to the virtual channel is Logic Pro. 

The issue can be resolved immediately by manually switching "buffer delay" and "enable small network..." back and forth. But it will eventually happen again after a while.

**Comment by dcorson-ticino-com** [comment]
>The issue can be resolved immediately by manually switching "buffer delay" and "enable small network..." back and forth. But it will eventually happen again after a while.

That is an indication that it is a network problem.
Clicking a buffer delay or the small network buffers ends the internet connection and then starts it again anew.

**Reply by hsshhhss** [reply]
Hi, Thanks for your explanation! I do consider this as an issue with Jamulus though, since I did not experience any network related issue in other softwares I use. I also notice that the audio issue becomes worse if I do not reset the connection manually. Do you think it is possible to add an "auto reconnect" feature when jitter is detected? Of course, whether to use it and the retry duration could be set by the user.

**Reply by dcorson-ticino-com** [reply]
> I did not experience any network related issue in other softwares I use

That is normal.  None of the other software that you use depends on very low latency times and the providers do not optimize for low latency either.  They look for high bandwidth, which is something else completely.  If your file download is fast with a 2 second latency you are still very happy, you do not even notice the very high latency.
We have this demand quite often, but you are better served by discussing our special usage case with your provider to get better service or changing providers to get lower latency on your network.  

(An aside, try using different servers, it can also be that the problem you are seeing is in the connection of the server and not your own connection at home.)

**Comment by ann0see** [comment]
Converting this to a discussion since it’s probably not an issue with the Jamulus code.
